PointOfDispensingApp/config.js

```diff
@@ -4,5 +4,16 @@ export default {
         UNDETERMINED: 0,
         VACCINATION_PROCEED_YES: 1,
         VACCINATION_PROCEED_NO: 2   
-    }	
+    },	
+
+    patientWorkflowState:
```

Comment:
I'd rename this to like `activeWorkflowState` to reflect that whatever enum you're in is the state you're in

Reply:
I deleted this file when I incorporated the content into store.js with MVS-521.  However, I changed "workflowState" to "activeWorkflowState" in store.js.

Reply:
On second thought, I'll keep that enum, but just put it in store.js!

Reply:
Sounds good!